aiohttp_json_rpc/rpc.py

```diff
@@ -294,12 +294,12 @@ def filter(self, topics):
             raise ValueError
 
         self.state[topic] = data
-
+        msg = None
```

Comment:
`msg = encode_notification(topic, data)` is cleaner. And i would rename `msg` to `notification`
Besides this, the patch seems good to me! :+1:

Reply:
I was trying to even not to encode notification when there's no client connected.

Reply:
ah ok this makes sense